### PR TITLE
Add seshat

### PR DESCRIPTION
**Pull request by clintval**
Adding the command line and Python package [seshat](https://github.com/clintval/tp53) for TP53 variant annotation of VCF files. This command line tool is a multi-language tool in which two subcommands are implemented in Python and the remaining command implemented in Scala. I opted for co-install of the software instead of complicating things with a meta-package at this time.

### CODE REVIEW

**Comment by clintval** [comment]
@BiocondaBot please add label

File: recipes/seshat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+about:
+  home: https://github.com/clintval/tp53
+  license: MIT
+  license_family: MIT
+  summary: Tools for programmatically annotating VCFs with the Seshat TP53 database.
+  doc_url: https://github.com/clintval/tp53
+
+extra:
+  additional-platforms:
+    - osx-arm64
```
Comment (martin-g) [inline recipes/seshat/meta.yaml:63]: No need of additional platforms for noarch recipe

File: recipes/seshat/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    folder: seshat
+  - url: https://pypi.io/packages/source/t/tp53/tp53-{{ version }}.tar.gz
+    sha256: 8efc0ec2b6b1a20b203c76ee3b41b02ead9de6e136dcedc695c2f7a595355574
+    folder: tp53
+
+build:
+  noarch: generic
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('seshat', max_pin="x") }}
```
Comment (martin-g) [inline recipes/seshat/meta.yaml:19]:
```suggestion
    - {{ pin_subpackage('seshat', max_pin="x.x") }}
```

**Comment by clintval** [comment]
Thanks @martin-g, I believe I've suited your review!

**Review by martin-g — APPROVED** [review]

**Pull request merged**